scout/server/blueprints/institutes/templates/overview/phenomodel.html

```diff
@@ -0,0 +1,363 @@
+  <div class="row d-flex align-items-center">
+    <div class="form-group col-4">
+      <input type="text" class="typeahead_omim form-control" name="omim_term" id="omim_term" data-provide="typeahead" autocomplete="off" placeholder="Search OMIM term..">
+    </div>
+    <div class="form-group col-5">
+      <input type="text" class="form-control" name="omim_custom_name" placeholder="Custom checkbox name (optional)">
+    </div>
+    <div class="form-group col-3">
+      <button type="submit" name="add_omim" class="btn btn-secondary btn-sm">Add OMIM term</button>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro hpo_checkbox_macro() %}
+  <div class="row d-flex align-items-center">
+    <div class="form-group col-4">
+      <input type="text" class="typeahead_hpo form-control" name="hpo_term" id="hpo_term" data-provide="typeahead" autocomplete="off" placeholder="Search HPO phenotype..">
+    </div>
+    <div class="form-group col-4">
+      <input type="text" class="form-control" name="hpo_custom_name" placeholder="Custom name checkbox name (optional)">
+    </div>
+    <div class="form-group col-2">
+      <input type="checkbox" class="form-check-input" id="includeChildren" name="includeChildren">
+      <label class="form-check-label" for="includeChildren">Include term children</label>
+    </div>
+    <div class="form-group col-2">
+      <button type="submit" name="add_hpo" class="btn btn-secondary btn-sm">Add HPO term</button>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro term_title(term_type) %}
+  <div class="row d-flex align-items-center">
+    <div class="form-group col-4">
+    {% if phenomodel.subpanels %}
+      <select name="{{term_type}}_subpanel_id" class="form-control" required>
+        <option value="" disabled selected>Select panel</option>
+        {% for subpanel_id, subpanel in phenomodel.subpanels.items() %}
+        <option value="{{subpanel_id}}">{{subpanel.title}}</option>
+        {% endfor %}
+      <select>
+    {% endif %}
+    </div>
+    <div class="form-group col-2">
+      <input type="checkbox" id="{{term_type}}HasTitle" name="{{term_type}}HasTitle">
+      <label class="form-check-label" for="{{term_type}}HasTitle">term has a title</label>
+    </div>
+    <div class="form-group col-5">
+      <input type="text" class="form-control" name="{{term_type}}TermTitle" id="{{term_type}}TermTitle" placeholder="Term title" disabled>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro subpanels_macro()%}
+  {%if phenomodel.subpanels %}
+    {% for subpanel_id, subpanel in phenomodel.subpanels.items()%}
+      <div class="card">
+        <div class="card-header">
+          <strong>{{subpanel.title}}</strong>&nbsp;&nbsp;<text class="text-muted">{{subpanel.subtitle}}</text>
+        </div>
+        <div class="card-body">
+
+          <div class="row">
+            {% if "checkboxes" in subpanel %}
+              <table class="table table-striped">
+                {% if "checkboxes" in subpanel %}
```

Comment:
Don't think you need to check that again! 😉

Reply:
Fixed. Wow, sharp eye!